### DISCUSSION TITLE
Frontend

### DISCUSSION THREAD

**Discussion by support-upyog** [upyog/UPYOG#679] [General]
UPYOG Frontend related query should be posted here.

**Comment by KeertanaITL** [comment]
I'm encountering an issue with the OTP (One-Time Password) not being received during the sign-in process on the UpYog domain after completing the Bootstrap setup. Here's what I've done so far:

Completed the Bootstrap setup for UpYog.
Opened the domain to access the sign-in page.
Entered the required details and requested an OTP.
Unfortunately, the OTP is not being received. I've checked my network connectivity and ensured that my phone number/email is correct. I've also looked into my spam/junk folder but to no avail.

Has anyone else experienced this issue? What steps can I take to resolve this and successfully receive the OTP?
